### PR TITLE
Run spelling checks on CI

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - File .codespell-whitelist is not an EIP
 - File .travis-ci.sh is not an EIP
 - File .travis.yml is not an EIP
 - EIP 1015 is in state Deferred, not Draft or Last Call
 - EIP 1167 is in state Final, not Draft or Last Call
 - EIP 1283 is in state Final, not Draft or Last Call
 - EIP 145 is in state Final, not Draft or Last Call
 - Error checking file EIPS/eip-1485.md
 - EIP 162 is in state Final, not Draft or Last Call
 - EIP 165 is in state Final, not Draft or Last Call
 - Error checking file EIPS/eip-1895.md
 - Error checking file EIPS/eip-1922.md
 - EIP 3 is in state Deferred, not Draft or Last Call
 - EIP 5 is in state Superseded, not Draft or Last Call
 - Error checking file EIPS/eip-615.md
 - EIP 627 is in state Final, not Draft or Last Call
 - EIP 706 is in state Final, not Draft or Last Call
 - Error checking file EIPS/eip-712.md
 - Error checking file EIPS/eip-758.md
 - Error checking file EIPS/eip-823.md
 - Error checking file EIPS/eip-918.md
 - Error checking file EIPS/eip-998.md
 - File README.md is not an EIP
 - File assets/eip-1057/test-vectors.md is not an EIP
 - File index.html is not an EIP
 - EIP 1011 requires approval from one of (@djrtwo, @chihchengliang)
 - EIP 1057 requires approval from one of (@ifdefelse)
 - EIP 1066 requires approval from one of (@carchrae, @naumenkogs, @expede)
 - EIP 107 requires approval from one of (wighawag@gmail.com)
 - EIP 1077 requires approval from one of (@alexvandesande)
 - EIP 1078 requires approval from one of (@alexvandesande)
 - EIP 1080 requires approval from one of (bradleat@inkibra.com)
 - EIP 1081 requires approval from one of (@3esmit, mark.beylin@consensys.net, kevin.owocki@consensys.net)
 - EIP 1108 requires approval from one of (@zac-williamson, @shadowfiend)
 - EIP 1109 requires approval from one of (@jbaylina)
 - EIP 1123 requires approval from one of (@djrtwo, @pipermerriam, @njgheorghita, @gnidan)
 - EIP 1154 requires approval from one of (@cag)
 - EIP 1155 requires approval from one of (eric@enjin.com, @coinfork, @ac0dem0nk3y, james@turing-complete.com, pc@horizongames.net)
 - EIP 1186 requires approval from one of (simon.jentzsch@slock.it, christoph.jentzsch@slock.it)
 - EIP 1193 requires approval from one of (@maiavictor, @frozeman, @marcgarreau, @ryanio)
 - EIP 1202 requires approval from one of (@yingogobot, @xinbenlv, @evbots)
 - EIP 1261 requires approval from one of (@chaitanyapotti, @pb25193)
 - EIP 1271 requires approval from one of (@izqui, @phabc, @abandeali1, @catageek, @shrugs, @frangio)
 - EIP 1388 requires approval from one of (a@colourful.land, @james-sangalli)
 - EIP 1417 requires approval from one of (@chaitanyapotti, @pb25193)
 - EIP 1450 requires approval from one of (howard@startengine.com, david@startengine.com, @johnshiple)
 - EIP 1462 requires approval from one of (mk@atlant.io, js@atlant.io)
 - EIP 1470 requires approval from one of (@thec00n)
 - EIP 1484 requires approval from one of (shanehampton1@gmail.com, andychorlian@gmail.com, anurag.angara@gmail.com, @noahzinsmeister)
 - EIP 1491 requires approval from one of (@cohabo)
 - EIP 1571 requires approval from one of (@chfast, pawel@ethereum.org, @mariusvanderwijden, @andrealanfranchi)
 - EIP 1613 requires approval from one of (@alex-forshtat-tbk, @drortirosh, @yoav-tabookey)
 - EIP 1616 requires approval from one of (@fiiiu, @spalladino, @thegostep, @0age, @elopio)
 - EIP 1681 requires approval from one of (@holiman)
 - EIP 1702 requires approval from one of (@sorpaas)
 - EIP 1775 requires approval from one of (@danfinlay, @bunjin)
 - EIP 1930 requires approval from one of (@wighawag)
 - EIP 1973 requires approval from one of (@lerajk, @qinjian)
 - EIP 210 requires approval from one of (@vbuterin)
 - EIP 725 requires approval from one of (@tyleryasaka, @frozeman)

**Comment by holiman** [comment]
Go for it!

File: EIPS/eip-1775.md [hunk 1]
```diff
@@ -453,7 +453,7 @@ We did not find a list of BIP43 purpose code so here is what we could gather:
 | 80 | [BIP80](https://github.com/bitcoin/bips/blob/master/bip-0080.mediawiki) | Hierarchy for Non-Colored Voting Pool Deterministic Multisig Wallets |
 | 84 | [BIP84](https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki) | Derivation scheme for P2WPKH based accounts |
 | 535348 | [Ledger app ssh](https://github.com/LedgerHQ/ledger-app-ssh-agent/blob/master/getPublicKey.py#L49) | |
-| 80475047| [GPG/SSH Ledger](https://github.com/LedgerHQ/ledger-app-openpgp-card/blob/master/doc/developper/gpgcard3.0-addon.rst#deterministic-key-derivation)| |
+| 80475047| [GPG/SSH Ledger](https://github.com/LedgerHQ/ledger-app-openpgp-card/blob/master/doc/developer/gpgcard3.0-addon.rst#deterministic-key-derivation)| |
```
Comment (axic) [inline EIPS/eip-1775.md]: Typo.

File: EIPS/eip-1271.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ contract ERC1271 {
 }
 ```
 
-`isValidSignature` can call arbitrary methods to validate a given signature, which could be context dependent (e.g. time based or state based), EOA dependant (e.g. signers authorization level within smart wallet), signature scheme Dependant (e.g. ECDSA, multisig, BLS), etc.
+`isValidSignature` can call arbitrary methods to validate a given signature, which could be context dependent (e.g. time based or state based), EOA dependent (e.g. signers authorization level within smart wallet), signature scheme Dependent (e.g. ECDSA, multisig, BLS), etc.
```
Comment (frangio) [inline EIPS/eip-1271.md]:
```suggestion
`isValidSignature` can call arbitrary methods to validate a given signature, which could be context dependent (e.g. time based or state based), EOA dependent (e.g. signers authorization level within smart wallet), signature scheme dependent (e.g. ECDSA, multisig, BLS), etc.
```

**Comment by axic** [comment]
Rebased to fix the conflict introduced with #2044.

**Comment by axic** [comment]
@nicksavers master is failing spelling checks because this merged in after #1965, which has typos.

**Comment by axic** [comment]
Fixed it in #2047.

**Comment by Kaykutee** [comment]
`EIP 2251 changes require approval from [`"KimberlyGardner35@gmail.com"``] #2251

**Pull request merged**